src/runtimeMain/kotlin/com/epam/drill/test/agent/instrumentation/http/selenium/Selenium.kt

```diff
@@ -109,8 +109,9 @@ class Selenium : Strategy() {
                         hashMap.put($TEST_NAME_CALC_LINE);
                         ${DevToolsClientThreadStorage::class.java.name}.INSTANCE.${DevToolsClientThreadStorage::addHeaders.name}(hashMap);
                     } catch(Exception e) { e.printStackTrace();}
-                    addDrillCookies();
                 }
+                if(getCurrentUrl().equals("data:,")){ execute("get", $ImmutableMap.of("url", $1)); }
```

Comment:
what is "data:," ? Is the page which is opened for the first time? It should be moved to variable